gtk4/src/subclass/widget.rs

```diff
@@ -926,7 +926,7 @@ pub unsafe trait WidgetClassSubclassExt: ClassStruct {
                 let action_name = action_name.to_owned();
                 let parameter_type = parameter_type.map(ToOwned::to_owned);
                 ctx.spawn_local(glib::clone!(@strong this, @strong action_name, @strong parameter_type, @strong activate => async move {
-                    activate(this, &action_name, parameter_type.as_ref()).await;
+                    activate(this, &action_name, parameter_type).await;
```

Comment:
Might also want to pass `action_name` in there directly instead of borrowing?

Reply:
Done